docs/tutorials/discordtut.mdx

```diff
@@ -0,0 +1,103 @@
+---
+title: Discord Tutorials
+---
+
+## Initial Setup
+
+Before starting your server, there are a few steps you have to take:
+
+1. First, you must create a new Discord bot via the [Discord Developer Portal](https://discord.com/developers/applications/).
+Once on that page, click on "New Application" button on the top right, give your bot a name, click on the checkbox to agree with the [Discord Developer ToS](https://discord.com/developers/docs/policies-and-agreements/developer-terms-of-service/) and [Developer Policy](https://discord.com/developers/docs/policies-and-agreements/developer-policy/) and then click "Create".
+> ![Creating Application](../../static/img/discord/create-app.gif)
+> `New Application` -> Give Application a Name -> `Create`
+
+2. Once you create the application, you'll be directed to its overview.
+From this screen, you'll need to copy your "Application ID" and save it for a later step. To copy your Application ID, click the upper-left most blue "Copy" button. Make sure to save it for a later step.
+> ![Copy Application ID](../../static/img/discord/copy-client-id.gif)
+> `Copy` -> Paste into Notepad for later step
+
+3. Optionally, you can set an icon for your application as it will be the icon for the bot too.
+> ![Avatar](../../static/img/discord/avatar.png)
+
+[//]: # (Deprecated)
+
+[//]: # (> ![Create Bot]&#40;../../static/img/discord/create-bot.gif&#41;)
```

Comment:
I'm not sure what these are here for? surely they can be removed?

Reply:
Resolved as of latest commit.